### PR TITLE
ENCD-5225-display-cropped-read-length-and-tolerance

### PR DESCRIPTION
**Pull request by khzlin**

### CODE REVIEW

File: src/encoded/schemas/file.json [hunk 1]
```diff
@@ -271,6 +271,15 @@
                 }
             }
         },
+        "cropped_read_length_tolerance": {
+            "comment": "Only bam files with cropped_read_length specified can have cropped_read_length_tolerance.",
```
Comment (zoldello) [inline src/encoded/schemas/file.json]: does "bam" supposed to be capitalized ("BAM")?

File: src/encoded/static/components/filegallery.js [hunk 2]
```diff
@@ -2815,6 +2815,20 @@ const FileDetailView = function FileDetailView(node, qcClick, auditIndicators, a
                         </div>
                     : null}
 
+                    {selectedFile.cropped_read_length !== undefined ?
```
Comment (zoldello) [inline src/encoded/static/components/filegallery.js:3129]: Why are you checking specifically for undefined?  Why not:

{selectedFile.cropped_read_length > 0 ?

File: src/encoded/static/components/filegallery.js [hunk 3]
```diff
@@ -2815,6 +2815,20 @@ const FileDetailView = function FileDetailView(node, qcClick, auditIndicators, a
...
+                        <div data-test="croppedreadlength">
+                            <dt>Cropped read length</dt>
+                            <dd>{selectedFile.cropped_read_length}</dd>
+                        </div>
+                    : null}
+
+                    {selectedFile.cropped_read_length_tolerance !== undefined ?
```
Comment (zoldello) [inline src/encoded/static/components/filegallery.js:3136]: Why are you checking specifically for undefined? Why not:

{selectedFile.ropped_read_length_tolerance > 0 ?

**Review by zoldello — CHANGES_REQUESTED** [review]
I made two comments and I suspect that they will result in changes.

File: src/encoded/static/components/filegallery.js (hunk 3 above)
Comment (khzlin) [reply src/encoded/static/components/filegallery.js:3136]: I am checking to see if cropped_read_length_tolerance has a value to be displayed.

File: src/encoded/schemas/file.json (hunk 1 above)
Comment (khzlin) [reply src/encoded/schemas/file.json]: I don't think it has to be.

**Review by zoldello — DISMISSED** [review]
Changes look good. My comments are nothing significant.

**Review by zoldello — APPROVED** [review]
This was a rebase on dev. I see the rebased commit entry in the history. A new demo was built.

**Pull request merged**